input/images/mcode-patient-bundle-pull.svg

```diff
@@ -1,20 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" contentScriptType="application/ecmascript" contentStyleType="text/css" height="159px" preserveAspectRatio="none" style="width:526px;height:159px;" version="1.1" viewBox="0 0 526 159" width="526px" zoomAndPan="magnify"><defs><filter height="300%" id="fumpyee5pov0f" width="300%" x="-1" y="-1"><feGaussianBlur result="blurOut" stdDeviation="2.0"/><feColorMatrix in="blurOut" result="blurOut2" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 .4 0"/><feOffset dx="4.0" dy="4.0" in="blurOut2" result="blurOut3"/><feBlend in="SourceGraphic" in2="blurOut3" mode="normal"/></filter></defs><g><text fill="#000000" font-family="Arial" font-size="18" lengthAdjust="spacingAndGlyphs" textLength="283" x="119" y="28.8838">mCODE Patient Bundle: Pull Model</text><line style="stroke: #A80036; stroke-width: 1.0; stroke-dasharray: 5.0,5.0;" x1="82" x2="82" y1="74.7969" y2="152.8784"/><line style="stroke: #A80036; stroke-width: 1.0; stroke-dasharray: 5.0,5.0;" x1="434" x2="434" y1="74.7969" y2="152.8784"/><rect fill="#FEFECE" filter="url(#fumpyee5pov0f)" height="30.0986" style="stroke: #A80036; stroke-width: 1.5;" width="150" x="5" y="39.6982"/><text fill="#000000" font-family="Arial" font-size="14" lengthAdjust="spacingAndGlyphs" textLength="136" x="12" y="59.8301">mCODE Data Sender</text><rect fill="#FEFECE" filter="url(#fumpyee5pov0f)" height="30.0986" style="stroke: #A80036; stroke-width: 1.5;" width="160" x="352" y="39.6982"/><text fill="#000000" font-family="Arial" font-size="14" lengthAdjust="spacingAndGlyphs" textLength="146" x="359" y="59.8301">mCODE Data Receiver</text><polygon fill="#A80036" points="93,101.9297,83,105.9297,93,109.9297,89,105.9297" style="stroke: #A80036; stroke-width: 1.0;"/><line style="stroke: #A80036; stroke-width: 1.0;" x1="87" x2="433" y1="105.9297" y2="105.9297"/><text fill="#000000" font-family="monospace" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="328" x="99" y="100.8638">GET [base]/Patient/[id]/$mcode-everything</text><polygon fill="#A80036" points="422,130.8784,432,134.8784,422,138.8784,426,134.8784" style="stroke: #A80036; stroke-width: 1.0;"/><line style="stroke: #A80036; stroke-width: 1.0; stroke-dasharray: 2.0,2.0;" x1="82" x2="428" y1="134.8784" y2="134.8784"/><text fill="#000000" font-family="Arial" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="295" x="89" y="130.1235">HTTP response containing mCODE Patient Bundle</text><!--MD5=[55c8d9f75a4e06b6afc17b107bb0bc7e]
-@startuml
-skinparam defaultFontName Arial
-
-title mCODE Patient Bundle: Pull Model
-hide footbox
-participant "mCODE Data Sender" as Sender
-participant "mCODE Data Receiver" as Receiver
-"Receiver" -> "Sender" : ""GET [base]/Patient/[id]/$mcode-everything""
-"Sender" - -> "Receiver" : HTTP response containing mCODE Patient Bundle

```

Comment:
Removed these `.svg` files because publisher.jar is now inserting the SVG inline into the HTML. It does this based on a naming convention: any files in `input/images-source` with a `.plantuml` extension get picked up by the publisher and can be inserted in a `.md` file with `<div>{%include some_filename.svg%}</div>`.